website-html/admin-community-update.html

```diff
@@ -26,7 +26,7 @@
 							|
 							<a href="admin-discord-requests.html" class="no-style more" class="no-style more">Discord Requests</a>
 							|
-							<a href="admin-discord-review.html" class="no-style more">Manage Discord</a>
```

Comment:
Are we changing this as well? I thought it was just the Discord Requests thing

Reply:
good catch, sloppy work here